.github/workflows/ci.yml

```diff
@@ -24,9 +23,9 @@ jobs:
           - test:twitter_utils:ios
           - test:parseui:all
           - test:parse_live_query:all
-          - package:release
```

Comment:
This removes `package:release`, is that for building the assets?

Reply:
Yes

Reply:
I'll add this as a note manually to the next release changelog.